docs/docs/install/script.md

```diff
@@ -2,15 +2,17 @@
 sidebar_position: 20
 ---
 
-# Install Script [Experimental]
+# Install script [Experimental]
 
 :::caution
 This method is experimental and not currently recommended for production use. For production, please refer to installing with [Docker Compose](/docs/install/docker-compose.mdx).
 :::
 
-:::note
-The install script only supports Linux operating systems and requires Docker to be already installed on the system.
-:::
+## Requirements
+
+Follow the [requirements page](/docs/install/requirements) to get started.
+
```

Comment:
```suggestion
The install script only supports Linux operating systems and requires Docker to be already installed on the system.
```

I think we need to keep this because it's not clear that the script is Linux ONLY. I do agree with mentioning to review the requirements first.